<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qGLMessageDialog</class>
 <widget class="QDialog" name="qGLMessageDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetNewAna" native="true">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>560</y>
     <width>711</width>
     <height>88</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;color: rgba(255,255,255,0.85);</string>
   </property>
   <widget class="QLabel" name="lblAdditionalMsg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>88</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lblMainMsg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>88</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoAll</set>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblMsgWithoutIcon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>88</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>800</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #525F7B;
border-radius: 8px 8px 0px 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblBg">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>800</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #525F7B;
    color: rgba(255,255,255,0.85);
border-top-left-radius: 8px;
border-top-right-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>800</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>401</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #363D4A;
color: rgba(255,255,255,0.85);
border-radius: 8px 8px 8px 8px;
border-top-right-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>430</y>
     <width>96</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnClose">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>350</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnOK">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>670</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnOK2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>670</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnCancel">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>670</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>widgetNewAna</zorder>
  <zorder>lblTitle</zorder>
  <zorder>lblBg</zorder>
  <zorder>lblIcon</zorder>
  <zorder>pBtnOK</zorder>
  <zorder>pBtnOK2</zorder>
  <zorder>pBtnCancel</zorder>
  <zorder>pBtnClose</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
